src/odemis/driver/test/npmc_test.py

```diff
@@ -41,23 +41,16 @@
 # arguments used for the creation of basic components
 CONFIG = {"name": "Delay Stage",
           "role": "delay-stage",
-          "port": "/dev/fake",
+          "port": "/dev/ttyUSB0",
           # "port": "/dev/ttyUSB0", Use this for the real hardware
           "axes": {
                 'x': {
                     'number': 1,
-                    'range': [-1, 1],
+                    'range': [0, 0.604644],
```

Comment:
I think one issue with this range is that if you go to the reference position with any negative offset, the reported position becomes immediately out of range. Generally speaking, it can happen with any range, as long as the axis gets near the range limit, but of course, with a range just at the reference position, you can immediately trigger issues.

If I understand correctly, the `updatePosition()` should work fine, but moveAbs() will get unhappy if you try to go back to a previously reported position. Maybe you should subtract the POS_COR offset before `checkMoveAbs()` so that all the possible actual range is allowed. However, it'll still feel strange as the axis range reported will not fit the position (but it's not possible to update the axis range in the current Odemis API). Could you try subtracting POS_COR offset when calling `checkMoveAbs()` and see if that helps?

Reply:
I'll ask about this once you're here.

Reply:
I think this is already done by removing the offset.

Reply:
Indeed.